codegen/templates/endpoint.tmpl

```diff
@@ -68,7 +68,7 @@ func New{{$handlerName}}(
 		{{ if len $middlewares | ne 0 -}}
 		zanzibar.NewStack([]zanzibar.MiddlewareHandle{
 			{{range $idx, $middleware := $middlewares -}}
-			{{$middleware.Name}}.NewMiddleWare(
+			deps.Middleware.{{$middleware.Name | pascal}}.NewMiddlewareHandle(
```

Comment:
the need to have pascal probably means we're going to run into convention errors at some point